apps/app-article/src/components/article-content.js

```diff
@@ -18,7 +19,13 @@ class Content extends Component {
             } else if (blockType === 'html') {
                 return this.renderHtml(block, key);
             } else if (blockType === 'factbox') {
-                return this.renderFactBox(block, key);
+                block.box.headline = 'Facta';
```

Comment:
Should be "FAKTA"

Reply:
Also, can you do this? If `box` does not exist in `block`,  this would not work.

Reply:
Could we just pass an object of
```
{
  box:
  {
    headline: "FAKTA",
    title: block.factBox.title,
    ... 
   }
}
```
to `renderGenericBox` ?